internal/pkg/provision/providers/firecracker/launch.go

```diff
@@ -118,6 +118,7 @@ func Launch() error {
 			cmd := firecracker.VMCommandBuilder{}.
 				WithBin("firecracker").
 				WithSocketPath(config.FirecrackerConfig.SocketPath).
+				WithArgs([]string{"--seccomp-level", "0"}).
```

Comment:
I'm not sure this is good change as it seems to disable secccomp filters altogether for firecracker

so this potentially makes it less secure/more vulnerable

Reply:
could it be that the problem is missing seccomp support? so that firecracker fails applying seccomp filters?

Reply:
I'll double check with a different version of firecracker

Reply:
ah, I see about `InstanceStart`, do you have the error message?

Reply:
So this is an issue with the Gentoo build of Firecracker, your intuition was spot on.